pkg/cloudprovider/provider/openstack/provider_test.go

```diff
@@ -102,9 +102,47 @@ const expectedBlockDeviceBootRequest = `{
   }
 }`
 
+const expectedBlockDeviceBootVolumeTypeRequest = `{
+	"server": {
+	  "availability_zone": "eu-de-01",
+	  "block_device_mapping_v2": [
+		{
+		  "boot_index": 0,
+		  "delete_on_termination": true,
+		  "destination_type": "volume",
+		  "source_type": "image",
+		  "uuid": "f3e4a95d-1f4f-4989-97ce-f3a1fb8c04d7",
+		  "volume_size": 10,
+		  "volume_type": "ssd",
```

Comment:
```suggestion
		  "volume_type": "ssd"
```

now it's too much of comas :)

Reply:
Either I am too tired or just to stupid, but isn't that the reverse of the above. 🤔 
https://github.com/kubermatic/machine-controller/pull/999#discussion_r650931706